HpToolsLauncher/Program.cs

```diff
@@ -125,5 +127,29 @@ private static void ShowHelp()
             Console.WriteLine("* For the details of the entire parameter list, see the online README on GitHub.");
             Environment.Exit((int)Launcher.ExitCodeEnum.Failed);
         }
+
+        private static void RunProcessFromUFTFolder(string[] args)
+        {
+            if (args[args.Count()-1] == "-origin")
+                return;
+
+            string toolLocation = Assembly.GetExecutingAssembly().Location;
+            if (!System.IO.File.Exists(toolLocation))
+                return;
+
+            string paramFileLocation = System.IO.Directory.GetCurrentDirectory();
+            string cmdLine = string.Join(" ", args, 0, args.Count());
+            cmdLine += " -origin";
+            int exitCode = 0;
+            if (!Environment.UserInteractive)
+            {
+                exitCode = ProcessExtensions.StartProcessFromUserSession(toolLocation, cmdLine, paramFileLocation, false);                
```

Comment:
Before starting process, write a output (use `Console.WriteLine`) to show we will start a new process in user session for diagnostics.